### PR TITLE
HEEDLS-609 : Move ConfigHelper to the data project

### PR DESCRIPTION
**Pull request by GeorgeBlackwell**
### JIRA link
_[HEEDLS-609](https://softwiretech.atlassian.net/browse/HEEDLS-609)_

### Description
_A technical debt ticket to move functionality from `ConfigHelper` to `ConfigurationExtensions` in the data project._

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [X] Run the formatter and made sure there are no IDE errors.
- [X] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [X] Manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme. Links (if any) are below:
  - [doc_1_here](link_1_here)
- [X] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [X] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

**Review by AlexJacksonDS — CHANGES_REQUESTED** [review]
There are some changes that I think should have been done that are not yet done, and some files are now in the wrong place following the changes.
Also a query about whether we've done the correct thing here for Steve.

**Comment by JBsoftwire** [comment]
I don't have any new comments to add to what Alex said, just to make sure you deal with the merge conflicts as well.

File: DigitalLearningSolutions.Data/Helpers/ConfigurationExtensions.cs [hunk 1]
```diff
@@ -91,5 +93,35 @@ public static string GetLearningHubSsoSecretKey(this IConfiguration config)
...
+
+        public static string GetDownloadSummaryUrl(this IConfiguration config, int progressId)
+        {
+            return $"{config[CurrentSystemBaseUrlName]}/tracking/summary?ProgressID={progressId}";
+        }
+
+        public static string GetConsolidationPathUrl(this IConfiguration config, string consolidationPath)
+        {
+            return $"{config[CurrentSystemBaseUrlName]}/tracking/dlconsolidation?client={consolidationPath}";
+        }
```
[the remark this answers is not in the document]
Comment (SteveJacksonSoft) [reply DigitalLearningSolutions.Data/Helpers/ConfigurationExtensions.cs:120]: That's a fair point, hadn't realised these existed. Putting them in a class dedicated to old system urls would make sense. `OldSystemEndpointHelper`?

[the remark this answers is not in the document]
Comment (GeorgeBlackwell) [reply DigitalLearningSolutions.Data/Helpers/ConfigurationExtensions.cs:120]: Agreed, @SteveJacksonSoft are you keen for this to be included in this piece of work?

**Review by AlexJacksonDS — APPROVED** [review]
I think this looks good now, you've just still got another merge conflict to resolve since the last merge from master

**Pull request merged**